src/routes/middlewares/error.ts

```diff
@@ -1,7 +1,16 @@
-import { Request, Response } from "express";
+import { Request, Response, NextFunction } from "express";
 import status from "http-status";
 
-export const errorHandler = (err: TypeError, _req: Request, res: Response) => {
+export const errorHandler = (
+  err: TypeError,
+  _req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  if (res.headersSent) {
```

Comment:
what is this change for?

Reply:
I had some issiue that i tried to solve, but not necesery anymore, return to original